### ISSUE TITLE
How to clear the data table. 

### ISSUE THREAD

**Issue by shakesber89** [haesleinhuepf/devbio-napari#39]
Hi
      Can you kindly let me know how to clear the varaible/values in the data table generate from shape features on  segmeneted image.

Best,
Jesu

**Comment by haesleinhuepf** [comment]
Hi @Jesuchristopher-Joseph ,

you can just close the table. What use-case do you have in mind? We could add a menu that "deletes" data more rigorously if I understand better what this could be good for.

Thanks!

Best,
Robert

**Comment by shakesber89** [comment]
Hi Robert 
                 Good day. Thanks for your reply. I am working on a cell-Stack image.   To indentify cell types.
Steps used:
1. Import stack. 2. Median filtering/smoothing, 3. Voronoi-Otsu Segment, 4. Generate region pros, 5. Generat U-Map  (comp :5 , dist : 0.2; Feature : mean intensity ) 6. To generte Cluster ID. When try to use U-MAp I can see five umap column. Umap-0 --- Umap4. Usuallylly you will get only two column  right. I dont know how to delet the colum in the table or how to limit the Umap - 2 column.

![image](https://user-images.githubusercontent.com/20415983/235742955-b8893cd1-b953-4709-a001-85e13ef789a3.png)

![umap errror](https://user-images.githubusercontent.com/20415983/235743081-edc4fb63-2f71-4297-8280-70fe07e06513.JPG)

**Comment by haesleinhuepf** [comment]
Ah I see. Give napari-spreadsheet a try! It gives you a lot of tools to modify tables! 
https://www.napari-hub.org/plugins/napari-spreadsheet

**Comment by haesleinhuepf** [comment]
> 1. I dont know how to delet the colum in the table or how to limit the Umap - 2 column.

On and one more thing: enter "comp: 2" when running the umap algorithm

**Comment by shakesber89** [comment]
Yes. I tried the comp with 2 .But still i get 5 columns. Is Napari -spread sheet is compatable with Devbio?

Thanks,
Jesu

**Comment by haesleinhuepf** [comment]
Sure, most napari plugins are compatible with each other:-)

**Comment by shakesber89** [comment]
Hi Robert you are right. I think there is a issue with the dimemsion reduction with UMap generation. Irrespective of the umap parameters ( n_neigbors , Min_dist) values you should get Umap_0 and Umap_1. As these contains the mapping coordinate values. Instead the  tool box it generatimg  Umap columns based on the N_neighbors . Is there any one who can fix it ?

Best,
Jesu

**Comment by haesleinhuepf** [comment]
Hi @Jesuchristopher-Joseph ,

I cannot reproduce your problem. I have just tried the UMAP dimensionality reduction algorithm as explained [here](https://github.com/BiAPoL/napari-clusters-plotter#usage) using these parameters:

![image](https://user-images.githubusercontent.com/12660498/235974394-6f198105-d975-4239-8a8a-17de4383cea2.png)

And it produced UMAP_0 and UMAP_1. 

Please [create an issue in the repository](https://github.com/BiAPoL/napari-clusters-plotter/issues) of the algorithm and be as precise as possible to allow the developers to reproduce your problem.
